src/config.py

```
@@ -0,0 +1,63 @@
+    """
+    # Load .env file if specified or default .env exists
+    if config_file:
+        load_dotenv(config_file)
+    else:
+        load_dotenv()
+
+    # Configuration dictionary to store loaded settings
+    config = {
+        'api_base_url': os.getenv('COINGECKO_API_BASE_URL', 'https://api.coingecko.com/api/v3'),
+        'api_key': os.getenv('COINGECKO_API_KEY', ''),
+        'request_timeout': int(os.getenv('COINGECKO_REQUEST_TIMEOUT', 30)),
```

Pylint (3.3.7)

[warning] os.getenv default type is builtins.int. Expected str or None.

(W1508)